runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxy.java

```diff
@@ -154,16 +151,15 @@ public CorfuCompileProxy(CorfuRuntime rt, UUID streamID, Class<T> type, Object[]
                 upcallTargetMap, undoRecordTargetMap,
                 undoTargetMap, resetSet);
 
-        metrics = CorfuRuntime.getDefaultMetrics();
```

Comment:
why MetricRegistry becomes non final?

Reply:
I thought local variable does not need to be final. Fixed.